lib/drand-client.js

```diff
@@ -0,0 +1,34 @@
+import {
+  fetchBeaconByTime,
+  HttpChainClient,
+  HttpCachingChain
+} from '../vendor/deno-deps.js'
+
+// See https://docs.filecoin.io/networks/mainnet#genesis
+const FIL_MAINNET_GENESIS_TS = new Date('2020-08-24T22:00:00Z').getTime()
+const FIL_MAINNET_BLOCK_TIME = 30_000 // 30 seconds
+
+/** @type {import('https://cdn.skypack.dev/drand-client@1.2.6/?dts').ChainOptions} */
+const DRAND_OPTIONS = {
+  // FIXME: beacon verification does not work when using drand-client via CDN :(
```

Comment:
How bad is this, and is there a way to fix it?

Reply:
> How bad is this

We are blindly trusting the endpoint https://api.drand.sh/ to provide honest responses. I'd say it's the usual web2 level of trust, where we trust our infrastructure providers and don't verify their responses.

> is there a way to fix it?

Sure. 
- Worst case: Rewrite the verification algorithm using Web Crypto instead of `@subtle/hash`.
- Best case: Find a CDN that can correctly bundle drand-client and all of its dependencies.

Reply:
Got it! And thanks for adding this as a comment. Did you check if there's a CDN that correctly bundles it?

Reply:
I tried cdn.skypack.dev. I like that this CDN does not minify/uglify the code, that's why I am using it for other dependencies. Unfortunately, it cannot bundle drand-client correctly.

Do you know about any other CDN I could try?

Reply:
I also think this is not preventing us from landing this pull request. Let's open a follow-up issue: https://github.com/filecoin-station/spark/issues/86